js/js_test.go

```diff
@@ -571,3 +571,20 @@ func TestUint8Array(t *testing.T) {
 		t.Errorf("Non-empty byte array is not externalized as a Uint8Array")
 	}
 }
+
+func TestInternalizeZeroVal(t *testing.T) {
+	type S struct {
+		*js.Object
+
+		string `js:"string"`
```

Comment:
Since this is a test of internalization logic (as I understand from our conversation in #617), I would suggest rewriting it to avoid using `js` struct field tags. I think it's simpler to use `Object.String` method, it's lower level and more explicit, making it more clear what the test is testing.